test/test_ops.py

```diff
@@ -388,6 +466,101 @@ def test_vmapvjp(self, device, dtype, op):
             for loop_out, batched_out in get_fallback_and_vmap_exhaustive(fn, args, {}):
                 self.assertEqual(loop_out, batched_out, atol=1e-4, rtol=1e-4)
 
+    # There are several variations we care about
+    # 1) primal batched (TODO)
+    # 2) tangent batched (batched grads) <--
+    # 3) both batched (TODO)
+    # The below tests (2) only.
+    @ops(functorch_lagging_op_db + additional_op_db, allowed_dtypes=(torch.float,))
+    @skipOps('TestOperators', 'test_vmapjvp', {
+        # These should to be fixed:
+
+        # Random failures: Copied from above
+        xfail('nn.functional.dropout'),  # randomness
+        xfail('double'),
+        xfail('linalg.householder_product'),
+
+        # Batching rule not implemented for aten::_local_scalar_dense
+        xfail('linalg.eigh'),
+        xfail('linalg.tensorinv'),
+        xfail('linalg.inv'),
+        xfail('linalg.matrix_power'),
+        xfail('linalg.cholesky'),
```

Comment:
We should note that jvp and vmap composition in pytorch core also has this problem

Reply:
This and the comment below, still don't see on the pytorch core side. Need to investigate further.

Reply:
Do you have a minimal repro of this for functorch that does not use the functorch test suite? If we have a minimal repro of this for a function (say, linalg.inv) using vmap and jvp we can easily convert it to a PyTorch core test case and then use that to determine if there is actually a problem in PyTorch core or not

Reply:
Oh wait I think I found the issue, see the latest update.